plugin/storage/integration/elasticsearch_test.go

```diff
@@ -139,14 +143,41 @@ func (s *ESStorageIntegration) esCleanUp(allTagsAsFields, archive bool) error {
 	return s.initSpanstore(allTagsAsFields, archive)
 }
 
-func (s *ESStorageIntegration) initSpanstore(allTagsAsFields, archive bool) error {
+func (s *ESStorageIntegration) initSamplingStore() error {
+	client, err := s.getEsClient()
+	if err != nil {
```

Comment:
One useful improvement we could make to these helper functions is to pass `t` all the way from the test entry point and just use `require.NoError(t, err)` instead of checks & return values.

Reply:
working on this one

Reply:
I added some changes in the last commit. please check